pages/accountLists/[accountListId].page.test.tsx

```diff
@@ -111,13 +112,33 @@ describe('AccountListsId page', () => {
   describe('AccountListId Redirects', () => {
     const restApiNotFoundErrorMessage = "Resource 'AccountList' is not valid";
 
+    const makeContext = async (
+      url: string = '/accountLists/[accountListId]/contacts',
+    ) => {
+      return (await getServerSideProps({
+        req: { url: url },
+        query: {
+          accountListId: 'account-list-1',
+        },
+      } as unknown as GetServerSidePropsContext)) as GetServerSidePropsReturn;
+    };
+
     it('replaces and redirects to the default account list id if Not Found', async () => {
       const query = jest.fn();
 
       (makeSsrClient as jest.Mock).mockReturnValue({
         query: query,
       });
-      query.mockRejectedValueOnce(new Error(restApiNotFoundErrorMessage));
+      query.mockRejectedValueOnce(
+        new ApolloError({
+          graphQLErrors: [
+            {
+              message: restApiNotFoundErrorMessage,
+              extensions: { code: 'NOT_FOUND' },
+            },
+          ],
+        } as unknown as ApolloError),
```

Comment:
Use `new GraphQLError()` to remove the need for a type cast.

```suggestion
        new ApolloError({
          graphQLErrors: [
            new GraphQLError(restApiNotFoundErrorMessage, {
              extensions: { code: 'NOT_FOUND' },
            }),
          ],
        }),
```